src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -288,14 +310,57 @@ public void createSnapshot(Snapshot snapshot, List<String> rowIds) throws Interr
         List<WalkRelationship> walkRelationships = WalkRelationship.ofAssetSpecification(asset);
         walkRelationships(datasetBqDatasetName, snapshotName, walkRelationships, rootTableId, projectId, bigQuery);
 
-        // create the views
-        List<String> bqTableNames = createViews(datasetBqDatasetName, snapshotName, snapshot, projectId, bigQuery);
+        snapshotViewCreation(datasetBqDatasetName, snapshotName, snapshot, projectId, bigQuery, bigQueryProject);
+    }
 
-        // set authorization on views
-        List<Acl> acls = convertToViewAcls(projectId, snapshotName, bqTableNames);
-        bigQueryProject.addDatasetAcls(datasetBqDatasetName, acls);
+    private static final String getAllLiveViewDataTemplate =
+        "INSERT INTO `<project>.<snapshot>.<dataRepoTable>` " +
+            "(<dataRepoTableId>, <dataRepoRowId>) " + // TODO is this line necessary?
+            "SELECT <dataRepoRowId>, '<tableId>' FROM `<project>.<dataset>.<liveView>`";
```

Comment:
I don't think we want a rowid table for a live view. It should show all of the data from the "live view" in the dataset. The whole idea is not to have to make a new snapshot every time a tidbit of data gets added to the dataset.